CMakeLists.txt

```diff
@@ -1,3 +1,11 @@
+OPTION(BUILD_SHARED_LIBS "Build the TorchCraft libraries shared." ON)
+
+IF(BUILD_SHARED_LIBS)
+  SET(TORCHCRAFT_LIBRARY_TYPE SHARED)
+ELSE()
+  SET(TORCHCRAFT_LIBRARY_TYPE STATIC)
+ENDIF()
```

Comment:
Wait! This should not be needed, cmake will do this automatically if you don't specify the library type. The option does not need to be declared at all, but we do it here so that we can have it *on* by default.

Reply:
Fixed